src/_posts/awoa-70.md

```diff
@@ -5,7 +5,6 @@ from: 2017-7-14
 date: 2017-7-21
 layout: blog
 awoa: true
-author: twitter|andgokevin|Kevin Ngo
```

Comment:
btw, filed this as issue #467 and opened PR #468

Reply:
haha PR race conditions

i removed the change here